packages/apps/web-viewer-test/.env

```diff
@@ -7,14 +7,30 @@ IMJS_APPLICATION_INSIGHTS_KEY=
 
 # ---- iTwin.js Authorization Client Settings for use in the "Use iTwin.js Auth Client" route. Defaults to General Purpose backend settings. If using a backend other than General Purpose, you would need to update the Viewer component's "backend" prop value from within the AuthClientHome.tsx component as well. ----
 
-IMJS_AUTH_CLIENT_ITWIN_ID="1bff8c44-3196-4231-b8f6-66cf6dacd45b"
-IMJS_AUTH_CLIENT_IMODEL_ID="71adc398-33bd-4ca9-9dec-fa6a74729bf6"
-IMJS_AUTH_CLIENT_IMODEL_ID2="435e704b-ae19-418c-a02e-fc75686e213c"
+# dev
+
+IMJS_AUTH_CLIENT_ITWIN_ID="5a8b90a4-3eb0-466a-a8c8-7dac58f37f65"
+IMJS_AUTH_CLIENT_IMODEL_ID="773b051b-1987-43e6-87e0-b07c18a18260"
+IMJS_AUTH_CLIENT_IMODEL_ID2="9083d310-8ad5-4fb8-908c-c86288e5ff9b"
+
+# prod
+
+IMJS_AUTH_CLIENT_ITWIN_ID_PROD="1bff8c44-3196-4231-b8f6-66cf6dacd45b"
+IMJS_AUTH_CLIENT_IMODEL_ID_PROD="71adc398-33bd-4ca9-9dec-fa6a74729bf6"
+IMJS_AUTH_CLIENT_IMODEL_ID2_PROD="435e704b-ae19-418c-a02e-fc75686e213c"
+
 IMJS_AUTH_CLIENT_CLIENT_ID=""
 IMJS_AUTH_CLIENT_REDIRECT_URI="http://localhost:3000/signin-callback"
 IMJS_AUTH_CLIENT_LOGOUT_URI="http://localhost:3000/logout"
 IMJS_AUTH_CLIENT_SCOPES ="imodelaccess:read imodels:read realitydata:read"
-IMJS_AUTH_AUTHORITY="https://ims.bentley.com"
+IMJS_AUTH_AUTHORITY="https://qa-ims.bentley.com"
```

Comment:
i _believe_ setting the url_prefix var should be enough to set the right authority so we dont need to manage this one ourselves